apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/welcome-tour/tour-steps.tsx

```diff
@@ -63,7 +63,8 @@ function getTourSteps(
 	localeSlug: string,
 	referencePositioning = false,
 	isSiteEditor = false,
-	themeName: string | null = null
+	themeName: string | null = null,
+	siteIntent: string | undefined
```

Comment:
![badge](https://img.shields.io/badge/review-nits-green.svg)

Now that we have 5 arguments, we might group some into an object.